tensorboard/plugins/profile/pod_viewer/details_card/details-card.ts

```diff
@@ -12,6 +12,11 @@ limitations under the License.
 
 namespace pod_viewer_details_card {
 
+interface StepBreakdownNode {
+  [key: string]: number;
```

Comment:
Hmm what would be an example of the key? Can there be a separate object for holding it?

Reply:
Added some examples.

Reply:
Done.